### PR TITLE
Clearify installation instructions related to gopls

### PR DESCRIPTION
**Pull request by bojan-embroker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -327,8 +327,10 @@ func Foo(bar, baz string) {}
 `gofumpt` is a replacement for `gofmt`, so you can simply `go get` it as
 described at the top of this README and use it.
 
-When using an IDE or editor with Go integrations, it's best to use `gofumpt` as
-part of `gopls`. The instructions below show how to do that for some of the
+When using an IDE or editor with Go integration based ono `gopls`,
```
Comment (bconway) [inline README.md:330]: Typo: "ono".

Comment (bojan-embroker) [reply README.md]: Fixed. Thank you!

**Comment by bconway** [comment]
Also, the [current documentation](https://code.visualstudio.com/docs/languages/go#_formatting) indicates that VS Code should use the following syntax:

```
"gopls": {
    "formatting.gofumpt": true
}
```

Not sure if that's worth updating while you're here.

**Comment by bojan-embroker** [comment]
> Also, the [current documentation](https://code.visualstudio.com/docs/languages/go#_formatting) indicates that VS Code should use the following syntax:
> 
> ```
> "gopls": {
>     "formatting.gofumpt": true
> }
> ```
> 
> Not sure if that's worth updating while you're here.

I do not use VS Code, so I don't feel comfortable making that change.

File: README.md [hunk 2]
```diff
@@ -344,8 +346,9 @@ the `gopls` settings, but they will still work.
 },
 ```
 
-#### Goland
+#### GoLand
 
+GoLand doesn't use `gopls` so it should be configured to used `gofumpt` directly.
```
Comment (dlsniper) [inline README.md:351]: Typo in `to used`, should be `to use`

**Review by mvdan — APPROVED** [review]
LGTM but please fix the typo that Florin pointed out :)

**Comment by bojan-embroker** [comment]
> LGTM but please fix the typo that Florin pointed out :)

Oh no, two typos! Fixed.

**Review by mvdan — APPROVED** [review]

**Pull request merged**